projects/minizinc/project.yaml

```diff
@@ -7,4 +7,3 @@ sanitizers:
   - address
   - memory:
      experimental: True
-  - undefined
```

Comment:
Why doesn't UBSAN work?

Reply:
We're currently unsure why it is not working, but it is failing the CI tests. We hope to add it back soon.